openedx/adg/common/custom_registration_form/models.py

```diff
@@ -0,0 +1,20 @@
+"""
+Custom registration app models.
+"""
+
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from openedx.adg.common.user_authn_override.constants import ADG_BUSINESS_LINES
+
+class ExtendedUserProfile(models.Model):
```

Comment:
How about `UserExtendedProfile`?

Reply:
I used this earlier but ExtendedUserProfile is more descriptive I think.